packages/manager/apps/web/client/app/domain/webhosting/order/steps/hosting/domain-webhosting-order-steps-hosting.controller.js

```diff
@@ -0,0 +1,39 @@
+import chunk from 'lodash/chunk';
+import sortBy from 'lodash/sortBy';
+
+import { OPTIONS_DETAILS } from './domain-webhosting-order-steps-hosting.constants';
+
+export default class {
+  /* @ngInject */
+  constructor(constants) {
+    this.constants = constants;
+  }
+
+  $onInit() {
+    this.showDetails = true;
+    this.availableOffers = chunk(
+      sortBy(
+        this.availableOffers.map((offer) => {
+          const order = OPTIONS_DETAILS
+            .find(({ label }) => label === offer.planCode);
+
+          Object.assign(
+            offer,
+            {
+              id: order ? order.id : null,
+              guideUrl: this.constants.urls.hosting[this.user.ovhSubsidiary],
+            },
+          );
+          return offer;
+        }),
+        'id',
+      ),
+      3,
```

Comment:
Why chunking by 3 ?

Reply:
For display matters

Reply:
I don't think that the way you handle your arrays in javascript should interfere with the way you want to render them in your html ... We have utilities such as bootstrap grids to handle those kind of display if I understand correctly what was the purpose

Reply:
Ok this was chunked because of `d-md-flex` class that was missing a `flex-wrap`. Fixing